### ISSUE TITLE
Add S.Port and FBus Telemetry

### ISSUE THREAD

**Issue by jasc76** [AlkaMotors/AM32-MultiRotor-ESC-firmware#136]
Hi there
can you add the possibility to output S.Port and FBus telemetry? It is a really nice thing to buy opensource esc's and have telemetry directly from the esc and AM32 sadly doesn't have this

**Comment by AlkaMotors** [comment]
Certainly.. additional telemetry protocols will be added. most likely to the experimental branch.

**Comment by parkflyer** [comment]
I would strongly support this! For many users FrSky telemetry was one of the main reasons to use BlHeli32…

**Comment by Voglberg** [comment]
Das würde ich auch unbedingt unterstützen.

**Comment by EmanuelStassar** [comment]
Here another FrSky user. I have several BLHeli32 ESC's and use the telemetry on all of them. So here is my +1 for adding it to the code. thank you some much! I can't code otherwise I would like to help out. I can test...

**Comment by jasc76** [comment]
ping ... any news on this function ?

**Comment by Voglberg** [comment]
Hello, so I'm also eagerly waiting for the S-Port telemetry support. Hopefully this will be implemented soon. It is always announced. Please, please.
